src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Triggers/BlobTriggerAttributeBindingProvider.cs

```diff
@@ -114,14 +57,14 @@ internal class BlobTriggerAttributeBindingProvider : ITriggerBindingProvider
             _loggerFactory = loggerFactory;
         }
 
-        public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext context)
+        public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext context)
```

Comment:
I found several methods like this that were async but not awaiting anything, so I fixed them

Reply:
The spots where `null` was returned when a Task is expected are also trouble! So thanks for fixing!